scripts/install_zds.sh

```diff
@@ -538,43 +585,48 @@ if  ! $(_in "-data" $@) && ( $(_in "+data" $@) || $(_in "+base" $@) || $(_in "+f
     # We check if ZMD is really up:
     nb_zmd_try=0
 
-    while ! curl -s $ZMD_URL > /dev/null && [ $nb_zmd_try -lt 40 ]
-    do
+    while ! curl -s "$ZMD_URL" >/dev/null && [ $nb_zmd_try -lt 40 ]; do
         sleep 0.2
-        nb_zmd_try=$(($nb_zmd_try+1))
+        nb_zmd_try=$((nb_zmd_try + 1))
```

Comment:
Ce changement est good ? Il faudra le Q/A soigneusement !

Reply:
Le `$` n’est pas nécessaire dans les contextes arithmétiques. Si la variable ne contient toujours qu’un seul nombre, le fonctionnement n’est pas modifié. Voir [ici](https://github.com/koalaman/shellcheck/wiki/SC2004).

Reply:
Ok c'est bon pour moi ! https://repl.it/repls/PoshTrainedRoutine